product_multi_company/models/product_category.py

```diff
@@ -8,22 +8,14 @@ class ProductCategory(models.Model):
     _inherit = "product.category"
 
     total_route_ids = fields.Many2many(
-        domain=lambda self: [
-            "|",
-            ("company_id", "=", False),
-            ("company_id", "in", self.env.companies.ids),
-        ]
+        domain=lambda self: [("company_id", "in", [False] + self.env.companies.ids)]
     )
 
     route_ids = fields.Many2many(
-        "stock.location.route",
+        "stock.route",
```

Comment:
Remove these overrides. That should go if ever done on a module `product_multi_company_stock`.

Reply:
All done! thanks